### PR TITLE
Enable fdk decoder for flatpak version

### PR DESCRIPTION
**Pull request by gal20**
Fixes missing audio in games that require AAC decoder. I tested it with Pokemon X and the music played correctly. Since flatpak runtimes are supposed to be consistent across different Linux distributions, it's unlikely to cause any incompatibility.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/5609)

### CODE REVIEW

**Comment by BreadFish64** [comment]
@liushuyu do you mind taking a look at this?

File: CMakeLists.txt [hunk 1]
```diff
@@ -41,7 +41,7 @@ CMAKE_DEPENDENT_OPTION(COMPILE_WITH_DWARF "Add DWARF debugging information" ON "
 
 option(USE_SYSTEM_BOOST "Use the system Boost libs (instead of the bundled ones)" OFF)
 
-CMAKE_DEPENDENT_OPTION(ENABLE_FDK "Use FDK AAC decoder" OFF "NOT ENABLE_FFMPEG_AUDIO_DECODER;NOT ENABLE_MF" OFF)
+option(ENABLE_FDK "Use FDK AAC decoder" OFF)
```
Comment (liushuyu) [inline CMakeLists.txt:44]: I would like to know why is this changed.

**Review by liushuyu — APPROVED** [review]
Overall LGTM

File: CMakeLists.txt (hunk 1 above)
Comment (gal20) [reply CMakeLists.txt:44]: Sorry, I didn't realize you can do -DENABLE_FDK=ON with cmake_dependent_option.
